client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCart.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {ReactElement, useState, useEffect} from 'react';
+import {useLocation, useNavigate} from 'react-router';
+import {EventRow} from './AdminPurchase';
+
+/**
+ * Math to dollar - `$${(Math.round(x * 100) / 100).toFixed(2)}`
+ *
+ * @param x
+ */
+const toDollar = (x: number) => `$${(Math.floor(x * 100) / 100).toFixed(2)}`;
```

Comment:
Why is this necessary. Does just doing `toFixed(2)` not work?

Reply:
Changed to x.toFixed(2).  Tested no issues.
![image](https://github.com/WonderTix/WonderTix/assets/97830813/6325d37a-3f25-4861-a286-7522c58911f8)
![image](https://github.com/WonderTix/WonderTix/assets/97830813/300c0577-a04c-4c0a-a36a-cdec713c99d8)